components/molecules/GamePlayedStory/GamePlayedStory.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import {Pressable, StyleSheet, Text, View, ViewStyle} from "react-native"
+import GameCard from '../GameCard/GameCard'
+import {FriendAvatar, FriendAvatarType} from '../FriendAvatar/FriendAvatar'
+
+export interface GamePlayedStoryProps {
+    title: string
+    style?: ViewStyle
+    gameBookImageUrl: string | undefined
+    gameBookTitle: string
+    onBookTapped: () => void
+    onStroyTapped: () => void
+    avatarImage: string | undefined
+    avatarUername: string
+    avatarStatus: FriendAvatarType
+}
+
+export const GamePlayedStory: React.FC<GamePlayedStoryProps> = (props) => {
```

Comment:
Let's follow the syntax convention that we're using within vizzable rather than this `React.FC` convention. I know that other things in lava-components are using that but I'd like us to move away from it